site/src/components/page-sidebar.tsx

```diff
@@ -462,31 +461,16 @@ export const Sidebar: VFC<SidebarProps> = ({
             opacity: 0,
           }}
         />
-        {pages.length > 1
```

Comment:
Not sure I understand why this change is necessary – could you explain?

Reply:
I'm not sure either actually, doesn't introduce any issues locally for me. I think I may have incorrectly thought it was causing an issue and then forget to revert the change, so I'm happy to revert it now.

Just want to clarify what this code was doing, as I'm still not entirely sure. We want pages that have a single sub-page to not display that sub-page directly, and instead render its sections directly? Or is there other reasoning for why we are special-casing this here?

Reply:
I don't actually know either – I didn't write this. Looks like it was introduced in https://github.com/blockprotocol/blockprotocol/commit/3ea6f3b4b09b9a2491e068ef7afc63f7db3a2e5d – any ideas?